runtime/web_worker.rs

```diff
@@ -56,29 +57,27 @@ impl WorkerId {
   }
 }
 
-type WorkerMessage = ZeroCopyBuf;
-
 /// Events that are sent to host from child
 /// worker.
-pub enum WorkerEvent {
-  Message(WorkerMessage),
+pub enum WorkerControlEvent {
```

Comment:
`Serialize` could probably be implemented here instead of `worker_host.rs` - I know you didn't change that, but I'd appreciate if you could sort this out